build-doc/ansible_module_documentation.py

```diff
@@ -112,6 +121,8 @@ def main():
                 doc, plainexamples, returndocs = module_docs.get_docstring(module_file_name)
```

Comment:
The length of the returned tuple change on version 2.3, it's failing for me.

The error is: `ValueError: too many values to unpack`

See the differences at:
2.3: https://github.com/ansible/ansible/blob/stable-2.3/lib/ansible/utils/module_docs.py#L147
2.2: https://github.com/ansible/ansible/blob/stable-2.2/lib/ansible/utils/module_docs.py#L133